bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1105,7 +1106,7 @@ Options getOptions() {
     void printLedgerMetadata(long ledgerId, LedgerMetadata md, boolean printMeta) {
         System.out.println("ledgerID: " + ledgerIdFormatter.formatLedgerId(ledgerId));
         if (printMeta) {
-            System.out.println(new String(md.serialize(), UTF_8));
+            System.out.println(new String(new LedgerMetadataSerDe().serialize(md), UTF_8));
```

Comment:
Because later I want to tweak behaviour based on what's in the ledger manager metadata and whether the client configuration says to store Ctime. The idea is that the managed ledger metadata contains a "maxMetadataFormatVersion", defaulting to 2 if absent. ManagedLedger reads this on boot (it's reading the znode anyhow), and uses it to decide which version to use to write new ledger metadata.  Version 3 will be binary metadata.

In this particular scenario, we should be doing md.toString(), but I didn't want to change that logic in what is primarily a refactor.